roles/ami_config/tasks/ami-tower/install.yml

```diff
@@ -83,4 +83,29 @@
     password: "{{ tower_password }}"
     body: '{{ lookup("file", tower_license) }}'
     body_format: json
+
+- name: Enable sudo escalation for Tower
+  lineinfile:
+    path: /etc/tower/settings.py
+    line: "AWX_PROOT_ENABLED=False"
+    backup: yes
+  become: true
+
+- name: Allow 'wheel' group to have passwordless sudo
+  lineinfile:
+    dest: /etc/sudoers
+    state: present
+    regexp: '^%wheel'
+    line: '%wheel ALL=(ALL) NOPASSWD: ALL'
+    validate: visudo -cf %s
+  become: true
+
+- name: Add sudoers users to wheel group
+  user:
+    name: "{{ item }}"
+    groups: wheel
+    append: yes
+  with_items: 
+    - awx
```

Comment:
Just curious, why `with_items` when there is only one item?

Reply:
just being flexible for future.  i can remove.